listenbrainz_spark/recommendations/train_models.py

```diff
@@ -69,8 +80,88 @@ def preprocess_data(playcounts_df):
     return training_data, validation_data, test_data
 
 
-def train(training_data, validation_data, num_validation, ranks, lambdas, iterations, alpha):
-    """ Train the data and get models as per given parameters i.e. ranks, lambdas and iterations.
+def generate_model_id():
+    """ Generate model id.
+    """
+    return '{}-{}'.format(config.MODEL_ID_PREFIX, uuid.uuid4())
+
+
+def get_best_model_path(model_id):
```

Comment:
I can confused by this name. why best when there are not evaluations of quality being made in this function?

Reply:
right. so initially we are training the model and getting the best one out of it. Later everywhere we are referring to that model. but I guess it's confusing.
So we should just say model path, right?

Reply:
Yes.